src/lib/rpc/config.ts

```diff
@@ -147,3 +147,20 @@ export async function saveActiveVersionChange(
     return false;
   }
 }
+
+export async function getBackgroundVideoDisabled(): Promise<boolean> {
+  try {
+    return await invoke("get_background_disabled");
+  } catch (e) {
+    exceptionLog("Unable to check if background is disabled with", e);
+    return null;
```

Comment:
your promise is a lie, this ain't a bool.  Pick a default (probably `false`) and return that instead